joblib/memory.py

```diff
@@ -518,6 +521,25 @@ def __reduce__(self):
         return (self.__class__, (self.func, self.store_backend, self.ignore,
                 self.mmap_mode, self.compress, self._verbose))
 
+    def in_store(self, *args, **kwargs):
```

Comment:
I would prefer a name that better reflects what Memory is about: caching results. Why not calling this function: `cached` ?
The concept of store is more related to the underlying way used to handle the cache.

Reply:
Sounds good to me, I'll rename the function to `cahced`.